src/curated-corpus/helpers/definitions.ts

```diff
@@ -67,31 +67,3 @@ export const curationStatusOptions: DropdownOption[] = [
 export type ApprovedItemFromProspect = Omit<ApprovedCorpusItem, 'language'> & {
   language: CorpusLanguage | undefined;
 };
-
-// scheduled surface guid to timezone map
-export const guidToUtcOffset = [
-  {
-    guid: 'NEW_TAB_EN_US',
-    timeZone: 'America/New_York',
-  },
-  {
-    guid: 'NEW_TAB_DE_DE',
-    timeZone: 'Europe/Berlin',
```

Comment:
are we removing these coz its handled in the server now, and client doesn't need to maintain these mapping? 

i remember an old PR in the server where we had explicit mapping from surfaceGuid to timezone: https://github.com/Pocket/curated-corpus-api/pull/661/files